README.md

```diff
@@ -70,6 +70,13 @@ To test your camera, go to the [**CONTROL** tab](https://docs.viam.com/fleet/con
 ## Build for local development
 
 The binary is statically linked with [FFmpeg v6.1](https://github.com/FFmpeg/FFmpeg/tree/release/6.1), eliminating the need to install FFmpeg separately on target machines.
+
+We support building this module using the Makefile for the following targets:
+|        | Linux | Android  | Darwin |
+|--------|-------|----------|--------|
+| arm64  | ✅    | ✅       | ✅     |
+| amd64  | ✅    | ❌       | ✅     |
```

Comment:
Have you confirmed that `amd64` `Darwin` actually works end to end? If not, maybe mark it as unknown / untested.

Reply:
Have not confirmed Darwin/amd64 - narked ❌ since it is untested.

Reply:
I think there is still some missing documentation here.

This table is currently only describes the targets. It does not describe the souces.

A user can provide `TARGET_OS` `TARGET_ARCH` and implicitly provide `SOURCE_OS` and `SOURCE_ARCH`. If I run `TARGET_OS=linux TARGET_ARCH=arm64 make` on a `Linux` `amd64` source I get go build errors (aka the make file doesn't tell me I'm doing something unsupported). I think we should document what are the supported & unsupported combos of `TARGET_OS`, `TARGET_ARCH`, `SOURCE_OS` and `SOURCE_ARCH`. 

Can you make the same `TARGET_OS`X`TARGET_ARCH` table for each supported `SOURCE_OS` X `SOURCE_ARCH` we intend to support building from?